example_cluster/README.md

```diff
@@ -0,0 +1,36 @@
+## Example Cluster
```

Comment:
Thanks 😄 

I was thinking that might be nice, or at least a link. Do I need to translate to RST? Maybe add to the installation section or the contribution section. I'd like a symlink in here too so that people can find the docs from here.